pithos/pandora/pandora.py

```diff
@@ -161,7 +161,7 @@ def json_call(self, method, args=None, https=False, blowfish=True):
                 raise PandoraError("Login Error", code, submsg="Invalid username or password")
             elif code == ApiError.LISTENER_NOT_AUTHORIZED:
                 raise PandoraError("Pandora Error", code,
-                    submsg="A Pandora One account is required to access this feature. Uncheck 'Pandora One' in Settings.")
+                    submsg="A Pandora Plus account is required to access this feature. Uncheck 'Pandora Plus' in Settings.")
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [E501] line too long (124 > 120 characters)

<!-- Comment by [SideCI](https://sideci.com) -->